### ISSUE TITLE
Proxy is not accepted with URL containing allowed special characters

### ISSUE THREAD

**Issue by VasilyVP** [CheshireCaat/browser-with-fingerprints#30]
If provide proxy as an URL containing allowed special characters it renurns error message.
Example of the such URL: `http://4klkj324kl2__cr.us;anon:l2oklkr23kl@proxyprovider.com`

Expected behavior: accept any allowed URL.

**Comment by dr3adx** [comment]
yes this is a problem and hope to see this fixed, its a high priority
